src/main/java/io/cdap/plugin/cdc/source/sqlserver/CTSQLServerConfig.java

```diff
@@ -58,42 +53,89 @@
   @Macro
   private final String username;
 
+  @Name(DBNAME)
+  @Description("Database Name")
+  public String dbName;
+
   @Name(PASSWORD)
   @Description("Password to use to connect to the specified database. Required for databases that " +
     "need authentication. Optional for databases that do not require authentication.")
   @Nullable
   @Macro
   private final String password;
 
+  @Name(SQN)
+  @Description("Need the CDC Sequence Number in the output schema")
+  @Nullable
+  @Macro
+  private final Boolean sqn;
+
+
+  @Name(CDCNUMBER)
+  @Description("CDC Sequence Number to start ingesting from")
+  @Nullable
+  @Macro
+  private final int cdcnumber;
+
+
+  @Name(TABLENAME)
+  @Description("Table name for CT Information")
+  @Nullable
```

Comment:
This is annotated as nullable but the code doesn't look like it treats it as nullable.

Reply:
agreed

Reply:
Will address this.